src/System.Reflection.MetadataLoadContext/src/System.Reflection.MetadataLoadContext.csproj

```diff
@@ -0,0 +1,161 @@
+    <Compile Include="System\Reflection\TypeLoading\Constructors\RoConstructor.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Constructors\RoDefinitionConstructor.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Constructors\RoSyntheticConstructor.cs" />
+    <Compile Include="System\Reflection\TypeLoading\CustomAttributes\CustomAttributeArguments.cs" />
+    <Compile Include="System\Reflection\TypeLoading\CustomAttributes\CustomAttributeHelpers.cs" />
+    <Compile Include="System\Reflection\TypeLoading\CustomAttributes\Ecma\EcmaCustomAttributeData.cs" />
+    <Compile Include="System\Reflection\TypeLoading\CustomAttributes\Ecma\EcmaCustomAttributeHelpers.cs" />
+    <Compile Include="System\Reflection\TypeLoading\CustomAttributes\RoCustomAttributeData.cs" />
+    <Compile Include="System\Reflection\TypeLoading\CustomAttributes\RoPseudoCustomAttributeData.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Events\Ecma\EcmaEvent.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Events\RoEvent.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Fields\Ecma\EcmaField.cs" />
+    <Compile Include="System\Reflection\TypeLoading\Fields\RoField.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Assignability.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\AssemblyNameData.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\CoreType.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\CoreTypes.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaDefaultValueProcessing.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaHelpers.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaModifiedTypeProvider.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaPinnedTypeProvider.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaResolver.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaSignatureTypeProviderForToString.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaToStringHelpers.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\EcmaWrappedTypeProvider.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\GuardedPEReader.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\InternalManifestResourceInfo.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\MetadataExtensions.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Ecma\MetadataTable.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\HashHelpers.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Helpers.cs" />
+    <Compile Include="System\Reflection\TypeLoading\General\Interop.netcoreapp.cs" Condition="'$(TargetGroup)' == 'netcoreapp'" />
```

Comment:
`Interop.Something.cs` is used exclusively for managed/native interop in CoreFX. I would call it something else - `ReflectionExtensions,netcoreapp.cs`, `Helpers.netcoreapp.cs`, ...

Reply:
OK - TypeExtensions.x.cs